src/rpc/core_rpc_server.cpp

```diff
@@ -703,31 +703,31 @@ namespace cryptonote
     if(!m_core.handle_incoming_tx(tx_blob, tvc, false, false, req.do_not_relay) || tvc.m_verifivation_failed)
     {
       res.status = "Failed";
-      res.reason = "";
+      std::string reason = "";
       if ((res.low_mixin = tvc.m_low_mixin))
```

Comment:
Why is this using an assignment instead of a comparison operator here and below?

Reply:
And why not use the new `get_text_reason` here? Might be related to my question before.

Reply:
Because if it did the res bools would end up uninitialized. They're supposed to be returned to the caller (wallet).

I'm not sure I understand the second question.